web/.storybook/vite.config.mts

```diff
@@ -1,9 +1,10 @@
 import { defineConfig } from 'vite';
 import { tsconfigPathsPlugin } from '@gravitational/build/vite/tsconfigPaths.mjs';
+import { reactPlugin } from '@gravitational/build/vite/react.mjs';
 
 export default defineConfig(() => ({
   // We don't use `reactPlugin` (which imports @vitejs/plugin-react-swc under the hood)
   // because Storybook doesn't support SWC compilter in Vite-based projects.
   // Once it's available, migrate from babel.
-  plugins: [tsconfigPathsPlugin()],
+  plugins: [tsconfigPathsPlugin(), reactPlugin('')],
```

Comment:
this was required for the styled-component `css` attribute to work

Reply:
Hmm, the comment says we _don't_ use `reactPlugin`, yet it's clearly used here. What don't I understand?

(Also: would you mind putting this statement directly into the code comment?)

Reply:
I'm not sure why this happens. Looks like `babel-plugin-styled-components` is not transforming the css prop (but I believe it should). I couldn't find anything useful in the storybook docs and issues, but I will keeping looking.

Reply:
i don't understand it either, i just removed the comment for now

Reply:
I gave up on that, I don't know why it doesn't work with the babel plugin alone. I only corrected the mode passed to the react plugin.